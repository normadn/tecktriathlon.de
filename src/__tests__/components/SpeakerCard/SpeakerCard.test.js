import React from "react";
import SpeakerCard from "../../../components/SpeakerCard";
import { shallow } from "enzyme";

describe("SpeakerCard", () => {
  it("renders correctly", () => {
    const speaker = {
      name: "Matthew Zeiler",
      title: "Founder and CEO of Clarifai",
      description:
        "Matthew is a machine learning Ph.D. and thought leader pioneering the field of applied artificial intelligence (AI). Matt's groundbreaking research in computer vision has propelled the image recognition industry from theory to real-world application. Matthew is a machine learning Ph.D. and thought leader pioneering the field of applied artificial intelligence (AI). Matt's groundbreaking research in computer vision has propelled the image recognition industry from theory to real-world application. Matthew is a machine learning Ph.D. and thought leader pioneering the field of applied artificial intelligence (AI). Matt's groundbreaking research in computer vision has propelled the image recognition industry from theory to real-world application. Matthew is a machine learning Ph.D. and ya.",
      image: "coolpic.png"
    };
    expect(shallow(<SpeakerCard {...speaker} />)).toMatchSnapshot();
  });
});
