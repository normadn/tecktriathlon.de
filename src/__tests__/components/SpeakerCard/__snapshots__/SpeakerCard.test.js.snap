// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpeakerCard renders correctly 1`] = `
<div
  css={
    Object {
      "@media(min-width: 820px)": Object {
        "paddingBottom": "112px",
      },
      "marginLeft": "auto",
      "marginRight": "auto",
      "maxWidth": "416px",
      "paddingBottom": "80px",
      "position": "relative",
      "zIndex": 3,
    }
  }
>
  <div
    css={
      Object {
        "background": "#ffffff",
        "borderRadius": "8px",
        "boxShadow": "0 4px 24px 0 rgba(185, 185, 185, 0.5)",
        "marginLeft": "13px",
        "marginRight": "13px",
      }
    }
  >
    <div
      css={
        Object {
          "borderTopLeftRadius": "inherit",
          "borderTopRightRadius": "inherit",
          "overflow": "hidden",
        }
      }
    >
      <div
        css={
          Object {
            "background": "url(coolpic.png) no-repeat center center",
            "backgroundSize": "cover",
            "borderTopLeftRadius": "inherit",
            "borderTopRightRadius": "inherit",
            "filter": "blur(0px)",
            "marginBottom": "0px",
            "minHeight": "300px",
            "transition": "1s",
            "width": "100%",
          }
        }
      />
    </div>
    <div
      css={
        Object {
          "lineHeight": "20px",
          "minHeight": "210px",
          "overflow": "hidden",
          "padding": "16px 24px 24px 24px",
          "position": "relative",
          "transition": "1s",
        }
      }
    >
      <h3
        css={
          Object {
            "color": "#00205b",
            "fontWeight": 800,
            "lineHeight": "28px",
            "textTransform": "uppercase",
          }
        }
      >
        Matthew Zeiler
      </h3>
      <h2
        css={
          Object {
            "fontWeight": 500,
            "lineHeight": "23px",
            "paddingTop": "8px",
          }
        }
      >
        Founder and CEO of Clarifai
      </h2>
      <div
        css={
          Object {
            "textOverflow": "hidden",
          }
        }
      >
        <p
          css={
            Object {
              "color": "#565656",
              "lineHeight": "1.43",
              "marginBottom": "-305px",
              "paddingBottom": "36px",
              "paddingTop": "24px",
              "textOverflow": "hidden",
              "transition": "1s",
            }
          }
        >
          Matthew is a machine learning Ph.D. and thought leader pioneering the field of applied artificial intelligence (AI). Matt's groundbreaking research in computer vision has propelled the image recognition industry from theory to real-world application. Matthew is a machine learning Ph.D. and thought leader pioneering the field of applied artificial intelligence (AI). Matt's groundbreaking research in computer vision has propelled the image recognition industry from theory to real-world application. Matthew is a machine learning Ph.D. and thought leader pioneering the field of applied artificial intelligence (AI). Matt's groundbreaking research in computer vision has propelled the image recognition industry from theory to real-world application. Matthew is a machine learning Ph.D. and ya.
        </p>
      </div>
    </div>
    <div
      css={
        Object {
          "paddingBottom": "24px",
          "paddingLeft": "24px",
          "paddingTop": "36px",
        }
      }
    >
      <SpeakerCardTextToggle
        defaultText="learn more"
        focused={false}
        focusedText="show less"
        onClick={[Function]}
      />
    </div>
  </div>
</div>
`;
